montecarlo/README.md

```diff
@@ -13,7 +13,7 @@ https://xecluster.lngs.infn.it/dokuwiki/doku.php?id=xenon:xenon1t:cmp:computing:
 
 **3)** ssh to:
 ~~~~
-login.xenon.ci-connect.net
+login.xenon.ci-connect.net or @login-el7.xenon.ci-connect.net
```

Comment:
I think we should delete the old login node here - hopefully the issue with the new host will be fixed really soon.